lib/tasks/temp.rake

```diff
@@ -290,4 +290,9 @@ namespace :temp do
     end
   end
 
+  desc 'Look for a fix requests with linebreaks in titles'
+  task :remove_linebreaks_from_request_titles => :environment do
+    InfoRequest.where("title LIKE ?", "%\n%").each { |request| request.save! }
```

Comment:
You've got `\r` in the test case – would that get caught by this?

I'm assuming this catches `\r\n`?